### PR TITLE
Update deps and changelog

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

**Review by nikku — COMMENTED** [review]
We could be a little bit more descriptive in the deps bump commit message :see_no_evil:

**Comment by barmac** [comment]
> We could be a little bit more descriptive in the deps bump commit message 🙈

I guess I would be more descriptive if it was about a single library :D

**Comment by nikku** [comment]
> > We could be a little bit more descriptive in the deps bump commit message 🙈
> 
> I guess I would be more descriptive if it was about a single library :D

Let me rephrase:

* I'd be more descriptive _or_
* I'd cluster things by domain just as dependabot does it

:smile:
